backend/app/conversations/unsubscribeConvo.js

```diff
@@ -12,10 +12,9 @@ function unsubscribeConvo(message) {
     }
     else {
       if (user.unsubscribed != true) {
-        unsubscribeAndAnonymizeUser(user)
         botReply(user, 'You got it. Just message us again if you ever change your mind!')
-      }
-      else {
+        unsubscribeAndAnonymizeUser(user)
```

Comment:
we should use a promise here to make sure nothing funky happens:
`  botReply(user, 'You got it. Just message us again if you ever change your mind!').then(() => { unsubscribeAndAnonymizeUser(user) })`